src/quo/components/links/internal_link_card/user/style.cljs

```diff
@@ -0,0 +1,71 @@
+(ns quo.components.links.internal-link-card.user.style
+  (:require [quo.foundations.colors :as colors]))
+
+(defn loading-circle
+  [theme]
+  {:background-color (colors/theme-colors colors/neutral-5 colors/neutral-80 theme)
+   :margin-right     4
+   :width            16
+   :height           16
+   :border-radius    16})
+
+(defn loading-first-line-bar
+  [theme]
+  {:background-color (colors/theme-colors colors/neutral-5 colors/neutral-80 theme)
+   :width            145
+   :height           16
+   :border-radius    6})
+
+(defn loading-second-line-bar
+  [theme]
+  {:background-color (colors/theme-colors colors/neutral-5 colors/neutral-80 theme)
+   :width            112
+   :height           8
+   :border-radius    6
+   :margin-bottom    17})
+
+(defn last-bar-line-bar
+  [theme]
+  {:background-color (colors/theme-colors colors/neutral-5 colors/neutral-80 theme)
+   :width            271
+   :height           16
+   :border-radius    6})
+
+(defn gradient-start-color
+  [customization-color theme]
+  (colors/theme-colors (colors/resolve-color customization-color theme 10)
+                       (colors/resolve-color customization-color theme 20)
+                       theme))
+
+(defn linear-gradient-props
+  [theme customization-color]
+  {:colors [(gradient-start-color customization-color theme) :transparent]})
+
+(defn container
+  [customization-color loading? theme]
+  (merge
```

Comment:
We don't need to `merge` here, just `assoc` into the result of calling `linear-gradient-props`.